scipy/special/xsf/specfun/specfun.h

```diff
@@ -5024,14 +5024,16 @@ void rmn2sp(int m, int n, T c, T x, T cv, int kd, T *df, T *r2f, T *r2d) {
     //          and joining factors
     // ======================================================
 
-    int k, j, j1, j2, l1, ki, nm3, sum, sdm;
+    int k, j, j1, j2, l1, ki, nm3;
     T ip, nm1, nm, nm2, su0, sw, sd0, su1, sd1, sd2, ga, r1, r2, r3,\
-           sf, gb, spl, gc, sd, r4, spd1, spd2, su2, ck1, ck2;
+           sf, gb, spl, gc, sd, r4, spd1, spd2, su2, ck1, ck2, sum, sdm;
```

Comment:
With `IMPLICIT DOUBLE PRECISION (A-H,O-Z)`,
`sum, sdm` start with `s`, and `s ∈ [O-Z]`.

https://github.com/scipy/scipy/blob/9d08eb58205fb669ff72c60080b18be61b14d8e1/scipy/special/specfun/specfun.f#L585-L599